setup.py

```diff
@@ -277,13 +305,13 @@ def read_requirements(path="requirements/run.txt"):
 setup(name=f'{NAPP_USERNAME}_{NAPP_NAME}',
       version=NAPP_VERSION,
       description='Store flow information and show statistics about them.',
-      url='http://github.com/amlight/flow_stats',
+      url='http://github.com/kytos-ng/flow_stats',
       author='FIU/AmLight team',
       author_email='ops@amlight.net',
       license='MIT',
-      install_requires=read_requirements(),
+      install_requires=read_requirements() + ["setuptools >= 59.6.0"],
```

Comment:
@rmotitsuki I think we'll need to add an explicit `packages=[]`, on setup.py to fully support py39. Even though it's passing, if you run a `pip install -r requirements/dev.txt` (when you're developing locally with python 3.9.12) you'd get this error as it is:

```
❯ pip install -r requirements/dev.txt
Obtaining file:///home/viniarck/repos/flow_stats (from -r requirements/dev.txt (line 7))
  Preparing metadata (setup.py) ... done
Obtaining kytos from git+https://github.com/kytos-ng/kytos.git#egg=kytos (from -r requirements/dev.txt (line 9))
  Cloning https://github.com/kytos-ng/kytos.git to ./.direnv/python-3.9.12/src/kytos
  Running command git clone --filter=blob:none --quiet https://github.com/kytos-ng/kytos.git /home/viniarck/repos/flow_stats/.direnv/python-3.9.12/src/kytos
  Resolved https://github.com/kytos-ng/kytos.git to commit 4d1ed9cec2a4eda26a0d032d768f1cc48c417d62
  Preparing metadata (setup.py) ... done
....

  Running setup.py develop for amlight-flow-stats
    error: subprocess-exited-with-error

    × python setup.py develop did not run successfully.
    │ exit code: 1
    ╰─> [16 lines of output]
        /home/viniarck/repos/flow_stats/.direnv/python-3.9.12/lib/python3.9/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Re
quirements should be satisfied by a PEP 517 installer.
          warnings.warn(
        error: Multiple top-level modules discovered in a flat-layout: ['settings', 'utils', 'match_fields', 'main'].

        To avoid accidental inclusion of unwanted files or directories,
        setuptools will not proceed with this build.

        If you are trying to create a single distribution with multiple modules
        on purpose, you should not rely on automatic discovery.
        Instead, consider the following options:

        1. set up custom discovery (`find` directive with `include` or `exclude`)
        2. use a `src-layout`
        3. explicitly set `py_modules` or `packages` with a list of names

        To find more information, look for "package discovery" on setuptools docs.
        [end of output]
```

This error is complaining that there are multiple top level modules, discovered. Since all NApps are loaded dynamically, and we're installing the dependencies with `install_requires` (for now), I think it's safe to set `packages=[]`. 

I've just tried it out locally with `packages=[]`, and `pip install -r requirements/dev.txt` worked, let me know if it also works locally for you, `tox` is also passing as expected. 

For `sdntrace` dependency it was resulting in the same error when running setup.py develop, so we'll need to add `packages=[]`, there as well, check out these branches/commits:

- [flow_stats branch `fix/pr_18_setup_py`](https://github.com/kytos-ng/flow_stats/compare/fix/pr_18_setup_py)

- sdntrace https://github.com/kytos-ng/sdntrace/commit/bc3888199cc6ab1f91c745ce6ef98efd9358ab35

Docker build also succeeded with these branches:

```
docker build -f Dockerfile --no-cache --build-arg branch_sdntrace=fix/setup_py --build-arg branch_flow_stats=fix/pr_18_setup_py  . -t amlight/kytos:dev2

Removing intermediate container 8225af7cb633
 ---> 403fbfa0cc42
Successfully built 403fbfa0cc42
Successfully tagged amlight/kytos:dev2
```